pynn_spinnaker/random.py

```diff
@@ -38,6 +39,11 @@ def __init__(self, host_rng, seed=None):
         assert host_rng is not None
         self._host_rng = host_rng
 
+        # Generate four word base seed for SpiNNaker RNGs
+        self.SeedWords = 4
+        self.base_seed = np.random.RandomState(seed=seed).randint(0x7FFFFFFF,
+                                          size=self.SeedWords).astype(np.uint32)
```

Comment:
The alignment seems a bit out here (dread at the thought of running flake8 is already kicking in) - maybe move all the parameters to ``randint`` onto the newline.